source/magpie/operation.d

```diff
@@ -0,0 +1,337 @@
+module magpie.operation;
+
+/*
+ * This file will contain work on operations of aggregate and join
+ * aggregate:   Operation to work on a row/ column of DataFrame to give results like mean/median/mode/elc
+ * merge:       It joins two DataFrame in a way similar to that in Pandas or any RDBMS software
+ */
+
+import magpie.dataframe: DataFrame;
+
+/++
+Description: Merges two DataFrames based on indexes
```

Comment:
what indices? this function takes what I presume are two data frames (add a constraint for them).

Reply:
Added the `DataFrame` constraints in 7015557